src/request.rs

```diff
@@ -89,6 +90,21 @@ impl<'a> Request<'a> {
         write!(fmt, r#"</methodCall>"#)?;
         Ok(())
     }
+
+    /// Serialize this `Request` into an XML-RPC struct that can be passed to
+    /// the [`system.multicall`](https://openacs.org/api-doc/proc-view?proc=system.multicall)
```

Comment:
This link should rather point to https://mirrors.talideon.com/articles/multicall.html - a mirror of the original XML-RPC mailing list discussion I think.

The OpenACS link made me think this was an API specific to that project, which I wouldn't really want to support. But given that this seems to be a relatively well-known XML-RPC extension (like `<int64>` and `<nil/>` values), I think this is fine.